scripts/umockdev-wrapper.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+''' Read a YAML file passed as the --config argument and execute the command
+supplied in a testbed configured with UMockdev. '''
+
+import argparse
+import os
+from typing import Dict, TextIO
+
+import gi
+import yaml
+
+try:
+    gi.require_version('UMockdev', '1.0')
+except ValueError as exc:
+    raise RuntimeError('Package gir1.2-umockdev-1.0 is required') from exc
+
+from gi.repository import UMockdev
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--config', type=argparse.FileType(), required=True)
+    parser.add_argument('command', help='Command to execute')
+    parser.add_argument('args', nargs=argparse.REMAINDER,
```

Comment:
Pros:
* semantically, nargs=+ on command means we're repeating a command, which shows in the usage as `command [command ...]`

Cons:
* code is slightly more verbose
* argparse.REMAINDER was removed from the documentation because of https://bugs.python.org/issue17050 - which feels like a good reason not to use this approach.

However, I think we can get the best of both worlds by using nargs=* on args. I'll go ahead with this approach but we can still discuss.

Reply:
Yea, I guess it's the REMAINDER that was bugging me, I had looked at it recently and was curious why it was growing less documented over time.

The implemented solution feels safe.